src/Instrumentation/ExtraeLocalManager.jl

```diff
@@ -0,0 +1,57 @@
+"""
+ExtraeLocalManager.jl
+Implements a copy of the default LocalClusterManager that starts
+workers with an overdubbed event loop.
+
+"""
+
+struct ExtraeLocalManager <: ClusterManager
+    np::Int
+    restrict::Bool  # Restrict binding to 127.0.0.1 only
+end
+
+Base.show(io::IO, manager::ExtraeLocalManager) = print(io, "ExtraeLocalManager()")
+
+function Distributed.launch(manager::ExtraeLocalManager, params::Dict, launched::Array, c::Condition)
+    dir = params[:dir]
+    exename = params[:exename]
+    exeflags = params[:exeflags]
+    bind_to = manager.restrict ? `127.0.0.1` : `$(LPROC.bind_addr)`
+    cookie = cluster_cookie()
+
+    #hookline = """using Distributed; f = Base.open("hooked.txt", "w"); write(f, $(repr(string(cookie))))"""
+    hookline = """
+        using Distributed
+        using Extrae
+        using Cassette
+        
+        Cassette.overdub(Extrae.ExtraeCtx(), start_worker, $(repr(string(cookie))))
```

Comment:
Are both `repr` and `string` needed? Either use or another no?

Reply:
I see